### DISCUSSION TITLE
`vercel link scope/project`

### DISCUSSION THREAD

**Discussion by simonhaenisch** [vercel/vercel#4889] [Ideas] [answered]
> We are looking at different ways of solving this issue such as introducing now link to create a project without deploying (today you have to deploy with now before developing locally which seems backwards).

(https://github.com/vercel/vercel/discussions/3991#discussioncomment-2943)

The fact that you have to create a development deployment just so you can link a project is definitely backwards 😅 

Can we get a `vercel link` command? It could maybe suggest the scope/project I most likely want to link (similar to how it happens with `vercel` after selecting the scope). It would also be great if it was possible to specify the project as a parameter, i. e. `vercel link <scope>/<project>` (good for "get started" sections of readmes).

Additionally, at the end it could ask whether the env should be pulled? (`vercel env pull`)

**Comment by styfle** [comment, marked as answer]
Indeed, `vercel link` is coming soon https://github.com/vercel/vercel/pull/4897

**Reply by styfle** [reply]
The canary channel has been updated with a new command, `vercel link`.

Additionally, `vercel dev` will ask to link to a project if it is not already.

Try it out now with `npm i -g vercel@canary`, thanks!
